workflow/airflow_libero/Dockerfile

```diff
@@ -0,0 +1,53 @@
+FROM python:3.6-slim
+
+ENV LANG=C.UTF-8
+
+ARG AIRFLOW_VERSION=1.9.0
+ARG AIRFLOW_HOME=/usr/local/airflow
+
+WORKDIR ${AIRFLOW_HOME}
+
+COPY Pipfile ${AIRFLOW_HOME}
+COPY Pipfile.lock ${AIRFLOW_HOME}
+COPY script/entrypoint.sh /entrypoint.sh
+COPY config/airflow.cfg ${AIRFLOW_HOME}/airflow.cfg
+
+RUN set -ex \
+    && buildDeps=' \
+        python3-dev \
+        libkrb5-dev \
+        libsasl2-dev \
+        libssl-dev \
+        libffi-dev \
+        build-essential \
+        libblas-dev \
+        liblapack-dev \
+        libpq-dev \
+        git \
+    ' \
+    && apt-get update -yqq \
+    && apt-get upgrade -yqq \
+    && apt-get install -yqq --no-install-recommends \
+        $buildDeps \
+        python3-pip \
+        python3-requests \
+        libmysqlclient-dev \
+        apt-utils \
+        curl \
+        rsync \
+        netcat \
+        locales \
+    && sed -i 's/^# en_US.UTF-8 UTF-8$/en_US.UTF-8 UTF-8/g' /etc/locale.gen \
+    && locale-gen \
+    && update-locale LANG=en_US.UTF-8 LC_ALL=en_US.UTF-8 \
+    && useradd -ms /bin/bash -d ${AIRFLOW_HOME} airflow \
+    && pip install -U pipenv \
+    && pipenv install --system --deploy
```

Comment:
Should be possible to clean up the build steps, but not worth optimising here.

Reply:
Yes agreed. I have already stripped out a lot but it could be further refined moving forward.